utils/handleFile.ts

```diff
@@ -0,0 +1,30 @@
+import { Request, Express } from 'express';
+import multer from 'multer';
+import mime from 'mime';
+import path from 'path';
+import { v4 as uuid } from 'uuid';
+
+export const storageDir = () => {
+  return path.join(__dirname, '../storage');
```

Comment:
'../' przed storage mają przenosić do katalogu wyżej?

Reply:
Tak żeby być w głównym katalogu, g7-headhunter-server/storage/css/nazwapliku.csv
Jak wrzucałem końcowy projekt na server to było trzeba dodać jeszcze ../../storage...